python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -81,8 +82,11 @@ class _ResizeChannel:
     resize its own backing store. The class instance is sent through the channel.
     """
 
-    def __init__(self, reader_ref: "ray.ObjectRef"):
-        self._reader_ref = reader_ref
+    def __init__(
+        self,
+        _reader_refs: Dict[str, "ray.ObjectRef"] = None,
```

Comment:
nit: `Optional[Dict[str, "ray.ObjectRef"]]`

could you add docstring? it's not apparent what's the key of the dict